spark/src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -562,6 +562,119 @@ class DeltaTable private[tables](
           TableFeatureProtocolUtils.FEATURE_PROP_SUPPORTED))
     toDataset(sparkSession, alterTableCmd)
   }
+
+  /**
+   *  Clone a DeltaTable to a given destination to mirror the existing table's data and metadata.
```

Comment:
```suggestion
   * Clone a DeltaTable to a given destination to mirror the existing table's data and metadata.
```

Could we repeat this for other functions as well? (To generate the doc)

Reply:
Updated